Linux cross build break for ARM

When cross building for ARM on x64 host,  following error is observed.

```
$ ./build-managed.sh -release -buildArch=arm -RuntimeOS=ubuntu.14.04 -BuildTests=True -SkipTests=True -BuildPackages=False -- /p:Outerloop=True
...
  Microsoft.XmlSerializer.Generator.Tests -> /opt/code/github/hqueue/corefx/bin/AnyOS.AnyCPU.Release/Microsoft.XmlSerializer.Generator.Tests/netcoreapp/Microsoft.XmlSerializer.Generator.Tests.dll
  /lib/ld-linux-armhf.so.3: No such file or directory
/opt/code/github/hqueue/corefx/src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj(57,5): error MSB3073: The command "/opt/code/github/hqueue/corefx/bin/testhost/netcoreapp-Linux-Release-arm/dotnet /opt/code/github/hqueue/corefx/bin/AnyOS.AnyCPU.Release/Microsoft.XmlSerializer.Generator.Tests/netcoreapp/dotnet-Microsoft.XmlSerializer.Generator.dll /opt/code/github/hqueue/corefx/bin/AnyOS.AnyCPU.Release/Microsoft.XmlSerializer.Generator.Tests/netcoreapp/Microsoft.XmlSerializer.Generator.Tests.dll /force /casesensitive" exited with code 255.
```

--- a/src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj
+++ b/src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj
@@ -6,7 +6,7 @@
     <DefineConstants>$(DefineConstants);XMLSERIALIZERGENERATORTESTS</DefineConstants>
   </PropertyGroup>
   <PropertyGroup>
-    <SkipTestsOnPlatform Condition="'$(TargetGroup)' == 'uap'">true</SkipTestsOnPlatform>
+    <SkipTestsOnPlatform Condition="'$(TargetGroup)' == 'uap' OR '$(ArchGroup)' == 'arm'">true</SkipTestsOnPlatform>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
     <!-- We're building netcoreap, run on the test CLI

